hail/src/main/scala/is/hail/expr/ir/NativeReaderOptions.scala

```diff
@@ -31,8 +28,31 @@ class NativeReaderOptionsSerializer() extends CustomSerializer[NativeReaderOptio
     })
 )
 
+object NativeReaderOptions {
+  def fromJson(optionsJV: JObject): NativeReaderOptions = {
```

Comment:
I really like this naming convention. I feel like I'm always hunting for how to (de)serialize an object.

Reply:
Thanks!  Yeah, I agree, the json serialization in particular seems uber-implicit.